sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/monitoring/MetricsIT.java

```diff
@@ -65,8 +89,50 @@ void shouldReturnHavePrometheusMetricsEndpoint() {
     assertThat(((Map<String, Object>) result.get("_links")).get("prometheus")).isNotNull();
   }
 
+  @Test
+  @Order(2)
```

Comment:
Would it be possible to reset the metrics before each test?
Would be cool to make each test independent.